first_order_model/shrink_util.py

```diff
@@ -1107,23 +1162,21 @@ def shrink_model(model_copy, layer_graph, layer, count, sort):
     """
     Return a new model with the layer corresponding to the argument layer shrunken by count outputs
     """
-    custom_deletions = []
+    filters_to_recursively_remove = []
     deleted_things = set()
-    deletions = compute_deletion(layer_graph, custom_deletions, deleted_things,
+    single_filter_details = compute_deletion(layer_graph, filters_to_recursively_remove, deleted_things,
```

Comment:
can you add a comment describing what this line does/how it sets the rest of the layer up for deletions?

Reply:
Added